src/seagulls/cli/_di_container.py

```diff
@@ -20,6 +20,7 @@
 from seagulls.examples.space_shooter import ShooterScene
 
 from ._framework import LoggingClient
+from ..examples._game_state import GameState
```

Comment:
you're importing a private file here. you just need to expose `GameState` in the `__init__.py` file and import from `seaglls.examples`.

Reply:
Should be resolved.